templates/CRM/Admin/Form/OptionGroup.tpl

```diff
@@ -26,8 +26,10 @@
               <td class="label">{$form.description.label}
             {if $action == 2}{include file='CRM/Core/I18n/Dialog.tpl' table='civicrm_option_group' field='description' id=$id}{/if}</td><td>{$form.description.html}</td>
           </tr>
+        {if !empty($form.name)}
         <td class="label">{$form.name.label}</td>
               <td>{$form.name.html}</td></tr>
```

Comment:
There's a closing `</tr>` tag in this if block that doesn't match anything else inside the block.

Reply:
Thanks @colemanw have fixed that up now hopefully

Reply:
Look up at line 20

Reply:
https://github.com/civicrm/civicrm-svn/commit/130a31d2e883414040f14974ee7d5226e0778eea#diff-8b58dbe61d9af355de0b73bc04bf4f8cb3e735a3df39c9192c02fa906ce969da

Reply:
Thanks @demeritcowboy fixed it properly now I would say

Reply:
Yes looks good. I can't see where this field comes up though - I think name is not exposed anymore.